README.md

```diff
@@ -1,11 +1,15 @@
 <!--
-Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
+ Copyright © 2021 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
why is this reverting the date back to 2021 only. Should it not be 2021-2022? probably in other places too, did not check every file. General rule of thumb is : if a file was created in 2022, then the header includes only 2022. Otherwise it includes "File Created"-"Current year" format